EIPS/eip-holdable-token.md

```diff
@@ -0,0 +1,293 @@
+| to | The address of the payee, to whom the tokens are to be paid if executed |
+| notary | The address of the notary who is going to determine whether the hold is to be executed or released |
+| value | The amount to be transferred. Must be less or equal than the balance of the payer. |
+| expires | `true` if the hold can expire, `false` if not |
+| expiration | The unix timestamp when the hold is expired |
+
+#### HoldExecuted
+
+Emitted when a hold has been executed.
+
+| Parameter | Description |
+| ---------|-------------|
+| holdIssuer | The address of the hold issuer of the hold |
+| operationId | The unique ID per hold issuer to identify the hold |
+| notary | The address of the notary who executed the hold |
+| heldValue | The amount which was put on hold during creation |
+| transferredValue | The amount which was used for the transfer |
+
+#### HoldReleased
+
+Emitted when a hold has been released.
+
+| Parameter | Description |
+| ---------|-------------|
+| holdIssuer | The address of the hold issuer of the hold |
+| operationId | The unique ID per hold issuer to identify the hold |
+| status | Can be one of the following values: `ReleasedByNotary`, `ReleasedByPayee`, `ReleasedOnExpiration` |
+
+#### HoldRenewed
+
+Emitted when a hold has been renewed.
+
+| Parameter | Description |
+| ---------|-------------|
+| holdIssuer | The address of the hold issuer of the hold |
+| operationId | The unique ID per hold issuer to identify the hold |
+| oldExpiration | The expiration time before the renewal |
+| newExpiration | The expiration time after the renewal |
+
+#### AuthorizedHoldOperator
+
+Emitted when an operator has been approved to create holds on behalf of another account.
+
+| Parameter | Description |
+| ---------|-------------|
+| operator | The address to be a operator of holds |
+| account | Address on which behalf holds will potentially be created |
+
+#### RevokedHoldOperator
+
+Emitted when an operator has been revoked from creating holds on behalf of another account.
+
+| Parameter | Description |
+| ---------|-------------|
+| operator | The address to be a operator of holds |
+| account | Address on which behalf holds could potentially be created |
+
+## Rationale
+
+This standards provides a functionality, to guarantee future payments, which is needed for many business cases where transfers have to be guaranteed.
+
+It goes a step further than the ERC-20 `approve` function by ensuring that the held balance will be available when the transfer is done. Something that can not be done with `approve`, as the approved amount is only a maximum spending amount, but never guaranteed to be available.
+
+## Backwards Compatibility
+
+This EIP is fully backwards compatible as its implementation extends the functionality of ERC-20.
+
+## Implementation
+The GitHub repository [IoBuilders/holdable-token](https://github.com/IoBuilders/holdable-token) contains the work in progress implementation.
```

Comment:
Can we add a readme to holdable interface reference project, to explain or link to the EIP

Reply:
I've added a basic README to the project